app/src/main/res/values/static_strings.xml

```diff
@@ -27,4 +27,7 @@
 
     <!-- GeckoView abbreviation used in AboutFragment -->
     <string name="gecko_view_abbreviation" translatable="false">GV</string>
+
+    <!-- Default title for pinned Pocket top site that links to trending Pocket site -->
+    <string name="pocket_pinned_top_articles" translatable="false">Top Articles</string>
```

Comment:
I don't think we'll need this translated because we are only exposing this feature to English locales

Reply:
@ekager Are there plans to bring this to other languages in the future or will this stay EN-only?

Reply:
In the first iteration the Pocket content we are linking to is only available in English. We will be adding other locales in the future to support more content.